From 4bbfb2b9f391f273744163cfda7bec96671e9d9c Mon Sep 17 00:00:00 2001
From: Florian Forster <octo@collectd.org>
Date: Thu, 18 Apr 2013 06:25:58 +0200
Subject: [PATCH 01/22] src/utils_mount.h: Add <stdio.h>.

This is required on Solaris 10. Thanks to dannypoo@ for reporting this
problem.

Github: #306
---
 src/utils_mount.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/utils_mount.h b/src/utils_mount.h
index 1f2403c..83f789b 100644
--- a/src/utils_mount.h
+++ b/src/utils_mount.h
@@ -26,6 +26,7 @@
 #ifndef COLLECTD_UTILS_MOUNT_H
 #define COLLECTD_UTILS_MOUNT_H 1
 
+#include <stdio.h>
 #if HAVE_FS_INFO_H
 # include <fs_info.h>
 #endif
-- 
1.9.3

